kitsune/sumo/static/sumo/js/forums.js

```diff
@@ -1,86 +1,86 @@
-/*global Marky, document, jQuery */
+/* global Marky:false, jQuery:false, template:false, gettext:false, KBox:false */
 /*
  * forums.js
  * Scripts for the forums app.
  */
 
-(function($){
+(function($) {
 
-    function init() {
-        Marky.createSimpleToolbar(
-            '.editor-tools',
-            '#reply-content, #id_content',
-            {mediaButton: true});
+  function init() {
+    Marky.createSimpleToolbar(
+      '.editor-tools',
+      '#reply-content, #id_content',
+      {mediaButton: true});
 
-        new k.AjaxPreview($('#preview'));
+    new k.AjaxPreview($('#preview')); // eslint-disable-line
```

Comment:
oh i'm guessing the linter doesn't like when you create objects and don't assign them?